templates/apiary-hms-readonly.json

```diff
@@ -1,4 +1,45 @@
 [
+  {
```

Comment:
probably can make this optional here by not including init container when using external database